client/src/components/content/RobotContainer/RobotContainer.jsx

```diff
@@ -4,9 +4,9 @@ import { Col, Row, Typography } from 'antd';
 import { PlayCircleOutlined, EditOutlined } from '@ant-design/icons';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
-import socket from '../../../utils/socket/socketConnections';
```

Comment:
oh my mistake. The socket import should actually be excluded because we don't need the startRobot function here anymore. I accidentally left the unused startRobot Function in RobotContainer. I fixed it all now. ✅